recipes/bioconvert/meta.yaml

```diff
@@ -1,46 +1,49 @@
 {% set name = "bioconvert" %}
-{% set version = "1.0.0.post0" %}
-{% set sha256 =  "bbc9c516873c337f33db9d698382b4e2dc341f47f5b6d226619614afa703faab" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "777b873596e8d0382e3b2540bcbcea8f381d27bd3a8e0f1b943a62d66c1fe025" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bioconvert-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
+  number: 0
   noarch: python
+  script: {{ PYTHON }} -m pip install . --use-pep517 -vv
   entry_points:
     - bioconvert=bioconvert.scripts.converter:main
     - bioconvert_init=bioconvert.scripts.init_convert:main
-  number: 0
+    - bioconvert_stats=bioconvert.scripts.stats:main
 
 requirements:
   host:
-    - python >=3
-    - setuptools
+    - python >=3.8
+    - pip
+    - cython
   run:
+    - python >=3.8
     - easydev
     - biosniff
     - colorlog
     - deeptools
-    - gffread
     - pandas
     - biopython >=1.70
     - mappy
     - matplotlib-base
     - networkx
-    - picard
```

Comment:
Similar here, Picard seems to be required by `bam2sam`. This could be switched to `picard-slim` which comes without R dependencies, since plotting is not required in this case.